src/utils/application.data.ts

```diff
@@ -161,3 +162,7 @@ const getIndexInApplicationData = (req: Request, appData: ApplicationData, key:
     throw createAndLogErrorRequest(req, `application.data getIndexInApplicationData - unable to find object in session data for key ${key} and ID ${id}`);
   }
 };
+
+export const getRemove = (appData: ApplicationData): Remove => {
```

Comment:
Fine, but just wondering why this was added as we don't have a similar function for Update.

Reply:
I was using `remove = appData.remove || {} ` in a few places so thought it would be good to have it one place with the `|| {}` to return an empty object instead of undefined. Happy to change it back if needed.

Reply:
No. All good I think.